### PR TITLE
 story Card

### PR DESCRIPTION
**Pull request by Nizar7zak**
Screenshot from 2021-11-04 04-27-22

### CODE REVIEW

**Review by modrayyes — APPROVED** [review]

**Review by reemsh2000 — APPROVED** [review]

**Review by NadiaJalalTamimi — APPROVED** [review]

File: client/src/components/common/Story/StoryCard.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import { Card } from 'antd';
+import PropTypes from 'prop-types';
+import { EditOutlined, DeleteOutlined } from '@ant-design/icons';
+
+import Img from '../Img';
+import 'antd/dist/antd.min.css';
```
Comment (LinaYahya) [inline client/src/components/common/Story/StoryCard.jsx:7]:
```suggestion
```

File: client/src/components/common/Story/StoryCard.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
```
Comment (LinaYahya) [inline client/src/components/common/Story/StoryCard.jsx:1]: rename file to index

File: client/src/components/common/Story/StoryCard.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                <DeleteOutlined />
+              </div>
+            </div>
+            <Img
+              src="https://www.aljazeera.net/wp-content/uploads/2021/04/12fg56.jpg?resize=770%2C513"
+              alt={storyInfo.title}
+              styleClass="card-img"
+            />
+
+          </>
```
Comment (LinaYahya) [inline client/src/components/common/Story/StoryCard.jsx:46]: remove fragment

File: client/src/context/CityContext.jsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import React, { createContext, useState, useEffect } from 'react';
+import { message } from 'antd';
+import PropTypes from 'prop-types';
+import { useLocation } from 'react-router-dom';
+import http from '../services/httpService';
+import { apiEndPoint } from '../services/config.json';
+
+export const CityContext = createContext();
+function CityProvider({ children }) {
+  const id = useLocation().pathname.split('/')[2];
```
Comment (LinaYahya) [inline client/src/context/CityContext.jsx:10]: check how to get id from params

File: client/src/pages/City.jsx [hunk 5]
```diff
@@ -1,9 +1,21 @@
...
+import { CityContext } from '../context/CityContext';
 
 function City() {
+  const { userData } = useContext(AuthContext);
+  const { cityData: cityInfo } = useContext(CityContext);
+  const { cityData, stories } = cityInfo;
+
   return (
-    <ScrollSpy />
+    <>
```
Comment (LinaYahya) [inline client/src/pages/City.jsx:12]:
```suggestion
    <div>
```

**Review by LinaYahya — CHANGES_REQUESTED** [review]

File: client/src/context/CityContext.jsx (hunk 4 above)
Comment (Nizar7zak) [reply client/src/context/CityContext.jsx:10]: done

File: client/src/pages/City.jsx (hunk 5 above)
Comment (Nizar7zak) [reply client/src/pages/City.jsx:12]: done

File: client/src/pages/City.jsx [hunk 6]
```diff
@@ -1,9 +1,21 @@
...
 function City() {
+  const { userData } = useContext(AuthContext);
+  const { cityData: cityInfo } = useContext(CityContext);
+  const { cityData, stories } = cityInfo;
+
   return (
-    <ScrollSpy />
+    <div>
+      {stories
+    && stories.length
```
Comment (LinaYahya) [inline client/src/pages/City.jsx:14]:
```suggestion
      {stories?.length
```
just as a suggestion 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining
check this link might be useful for you

**Review by LinaYahya — APPROVED** [review]

File: client/src/pages/City.jsx (hunk 6 above)
Comment (Nizar7zak) [reply client/src/pages/City.jsx]: done, Thank you

**Pull request merged**